DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@model SearchableDelegateViewModel
+@{
+  var activeTagCss = Model.IsActive ? "nhsuk-tag" : "nhsuk-tag nhsuk-tag--grey";

```

Comment:
Could we change this to green when active and red if inactive to show success/warning states? And maybe change the admin tag into grey as it's fairly neutral? (and update the ticket accordingly)